docker/Dockerfile

```diff
@@ -8,4 +8,5 @@ RUN \
       multiqc=1.15 \
    && micromamba clean -a -y
 
+ENV PATH /opt/conda/bin:$PATH
```

Comment:
should not be like this? 

https://github.com/seqeralabs/libseqera/blob/6a7efe04f47717acbdfae2db989d6e943fa4ece7/wave-utils/src/main/resources/templates/conda/dockerfile-conda-file.txt#L7-L7

Reply:
thanks good point! - done